### DISCUSSION TITLE
Why are device position limtited to Uxx.0 and Uxx.5?

### DISCUSSION THREAD

**Discussion by dokh** [netbox-community/netbox#17011] [General]
I'm wondering why the postion for devices is limited to .0 and .5 of a 1U?
If I'm not mistaken there are usually 3 holes per 1U so that means devices could be mounted on position .0, .33 and .66.

regards, Koen

**Comment by markkuleinio** [comment]
Note that the holes in the rack are not evenly distributed, so please never try to utilize racks in 1/3 increments.

Just my 0.05 €

**Reply by dokh** [reply]
True, true, ... but real life isn't always fair.

**Comment by jsenecal** [comment]
Here is the original discussion around that topic:
https://github.com/netbox-community/netbox/issues/51

You can also look at the other related issues for additional context clues

**Reply by dokh** [reply]
Indeed it seems the request/discussion has been made before.

I do understand the need for .5U increments for device height, but the device height is unrelated to the .33 increments between rail perforations.

Looking at answers from certain developers I have a feeling they are not really sympathetic to real world requests. I wonder if they actually manage networks?

**Reply by jsenecal** [reply]
I don't think your last comment will attract any sympathy from the dev team. Regardless, I think it's more about reaching 80% of the use cases whilst limiting the impacts on UI/UX and code complexity.
